.eleventy.js

```diff
@@ -1,5 +1,17 @@
 const { isDev } = require('./pages/views/_data/env');
-const markdown = require('markdown-it')({ html: true });
+
+const hljs = require('highlight.js');
+const markdown = require('markdown-it')({
+  html: true,
+  highlight: function (str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        return hljs.highlight(str, { language: lang }).value;
+      } catch (__) { }
+    }
+    return '';
+  }
+});
```

Comment:
Maybe markdown-it-highlightjs use instead?

Reply:
I had difficulties connecting it, today I will try again and come back with an answer

Reply:
I don't think this is a good idea. It's a super simple library, but it does not have enough support as I see.
So prefer to use our own code that we have access to instead of using third libraries.

Reply:
> Maybe markdown-it-highlightjs use instead?

Eleventy has its own syntax highlighting, but since we use shortcode {% markdown %}, we cannot use it. I  think highlight.js is the best solution now